internal/cmap/cmap.go

```diff
@@ -53,13 +101,18 @@ type CMap struct {
 	// For regular cmaps.
 	codespaces []Codespace
 
-	// For ToUnicode (ctype 2) cmaps.
-	codeToUnicode map[CharCode]rune
+	// Used by ctype 1 CMaps.
```

Comment:
both of those ctype 1?  Seems like the previous comment was referring to ctype2, is that not valid anymore?